src/Microsoft.ML.Tokenizers/Tokenizer.cs

```diff
@@ -35,7 +35,23 @@ public abstract class Tokenizer
         /// <param name="textSpan">The span of the text to encode which will be used if the <paramref name="text"/> is <see langword="null"/>.</param>
         /// <param name="settings">The settings used to encode the text.</param>
         /// <returns>The encoded results containing the list of encoded Ids.</returns>
```

Comment:
> /// <returns>The encoded results containing the list of encoded Ids.</returns> [](http://example.com/codeflow?start=7&length=79)

could you please add a remark here telling if performance is important, it is recommended to override and provide optimized implementation for this method. we need to add this remark to other methods we are providing default implantation for.